tests/e2e/specs/editor/media/insertMovVideo.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { percySnapshot } from '@percy/puppeteer';
+
+/**
+ * Internal dependencies
+ */
+import { createNewStory } from '../../../utils';
+
+const MODAL = '.media-modal';
+
+describe('Inserting .mov from dialog', () => {
+  // Uses the existence of the element's frame element as an indicator for successful insertion.
+  it('should not list the .mov', async () => {
+    await createNewStory();
+    await expect(page).not.toMatchElement('[data-testid="FrameElement"]');
+    await expect(page).toClick('button', { text: 'Upload' });
+
+    await page.waitForSelector(MODAL, {
+      visible: true,
+    });
+    const btnTab = '#menu-item-browse';
+    await page.waitForSelector(btnTab);
+    await page.evaluate((selector) => {
+      document.querySelector(selector).click();
+    }, btnTab);
+    const btnSelector =
+      '.attachments-browser .attachments .attachment:first-of-type';
+    await page.waitForSelector(btnSelector);
+    await expect(page).not.toMatchElement('.type-video.subtype-quicktime');
+    await percySnapshot(page, 'Avoid inserting .mov files');
+
+    const closeBtnSelector = '.media-modal-close';
```

Comment:
Why do you need these lines?

Reply:
the ones after the snapshot? I shouldn't have deleted my last comment :D If I don't close the media picker dialog after the snapshot the e2e tests fail with this error:
```
  ● Inserting .mov from dialog › should not list the .mov

    expect(received).toPassAxeTests(expected)

    Expected page to pass Axe accessibility tests.
    Violations found:
    Rule: "landmark-one-main" (Document must have one main landmark)
    Help: https://dequeuniversity.com/rules/axe/3.5/landmark-one-main?application=axe-puppeteer
    Affected Nodes:
      html
        Fix ALL of the following:
          - Document does not have a main landmark.
    
    Rule: "list" (<ul> and <ol> must only directly contain <li>, <script> or <template> elements)
    Help: https://dequeuniversity.com/rules/axe/3.5/list?application=axe-puppeteer
    Affected Nodes:
      #__attachments-view-51
        Fix ALL of the following:
          - List element has direct children with a role that is not allowed: checkbox.
    

      194 |   }
      195 | 
    > 196 |   await expect(page).toPassAxeTests({
          |                      ^
      197 |     // Temporary disabled rules to enable initial integration.
      198 |     disabledRules: [
      199 |       'aria-allowed-role',

      at runAxeTestsForStoriesEditor (config/bootstrap.js:196:22)
      at Object.<anonymous> (config/bootstrap.js:232:3)
```
I don't think I control this rendering (the thumbnails with checkboxes inside the media picker) so I simply closed the dialog.